libphobos/libdruntime/core/sys/posix/stdio.d

```diff
@@ -279,7 +279,7 @@ unittest
 { /* Note: open_memstream is only useful for writing */
     import core.stdc.string : memcmp;
     char* ptr = null;
-    char[] testdata = ['h', 'e', 'l', 'l', 'o', 0];
+    char[6] testdata = ['h', 'e', 'l', 'l', 'o', 0];
```

Comment:
Ok. If we are building all sources now, that could contribute towards larger binaries, unless we have smarter moduleinfo emission than I think.